tests/client/fixtures/files-transfer-unicode.json

```diff
@@ -42,15 +42,15 @@
         "filegrpuuid": "",
         "sip": null,
         "checksumtype": "sha256",
-        "originallocation": "%transferDirectory%objects/\u305f\u304f\u3055\u3093 directories/need sanitization/checking here/lion\u5199\u771f.svg",
+        "originallocation": "%transferDirectory%objects/\u305f\u304f\u3055\u3093 directories/need_name_change/checking here/lion\u5199\u771f.svg",
```

Comment:
This path should use spaces between the words instead of underscores, like the original.